src/lib/dacc.js

```diff
@@ -64,5 +120,28 @@ export const runDACCService = async client => {
       `${nMeasuresSent} measures sent to DACC for account ${account._id}`
     )
   }
+
   return shouldRestartService
 }
+
+/**
+ * Run the bike goal DACC service
+ *
+ * @param {import("cozy-client/dist/index").CozyClient}} client - The cozy-client instance
```

Comment:
tips: You can create a typedef: 

```
/**
 * @typedef {import("cozy-client/dist/index").CozyClient} CozyClient
 */
```

and then juste 
```
@param {CozyClient} client
```

so much easiiiier :D

Reply:
I don't know about easier, but more elegant for sure :)

Reply:
Easier since you don't have to look for the import and copy / paste it everywhere ^^.

Reply:
Ah, for me the IDE kindly auto-completes